src/modules/Bootloader.rb

```diff
@@ -146,10 +146,17 @@ def Read
         ::Bootloader::BootloaderFactory.current = ::Bootloader::BootloaderFactory.proposed
         ::Bootloader::BootloaderFactory.current.propose
       rescue ::Bootloader::BrokenConfiguration => e
-        ret = Yast::Report.AnyQuestion(_("Broken Configuration"),
+        msg = if e.is_a?(::Bootloader::BrokenConfiguration)
```

Comment:
This looks strange with the `rescue ::Bootloader::BrokenConfiguration` statement above, `e` cannot be something else. Or is `::Bootloader::UnsupportedOption` missing in the `rescue`?

Reply:
yes, it is missing.